src/materials/MeshDepthMaterial.js

```diff
@@ -46,6 +46,10 @@ function MeshDepthMaterial( parameters ) {
 	this.wireframe = false;
 	this.wireframeLinewidth = 1;
 
+	// far clipping plane in both RGBA and Basic encoding
+	this.clearColor = new THREE.Color( 1.0, 1.0, 1.0 );
+	this.clearAlpha = 1.0;
```

Comment:
I am not sure `ClearColor` is an appropriate property for a material.

Just pass in the clear color to `SAOPass.renderOverride()`.